src/shared/components/Tabs/Tabs.styles.tsx

```diff
@@ -22,7 +22,8 @@ export const StyledTabs = styled(Tabs, shouldForwardProp)`
     uiType === UiType.Primary &&
     `
       width: 90rem;
-      margin: 0 auto;
+      margin: 0;
+      margin-left: 2rem;
```

Comment:
Have to mention that the Tabs is the dependency for other components, please make sure if they display correctly :
EventForm
Feedback
BuilderActivity
BuilderActivityFlow
BuilderApplet
Applet
Main
RespondentData